mirror/src/main/java/net/runelite/client/plugins/mirror/MirrorPlugin.java

```diff
@@ -52,16 +56,32 @@
 	@Inject
 	private Client client;
 
+	@Inject
+	private MirrorConfig config;
+
+	@Provides
+	MirrorConfig getConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(MirrorConfig.class);
+	}
+
 	public static JFrame jframe;
 	public static final Canvas canvas = new Canvas();
 	public static BufferedImage bufferedImage;
+	private static String title = "OpenOSRS Mirror";
+
+	private void updateTitle()
```

Comment:
make this update the title automatically so you dont need the class variable `title`